apps/slack/frontend/src/useConnect.ts

```diff
@@ -34,8 +34,15 @@ export const useConnect = () => {
 
           await apiClient.createAuthToken(sdk, cma, token, installationUuid);
         } catch (e) {
-          console.error(e);
-          sdk.notifier.error(genericErrorMessage);
+          const error = e as Error;
+
+          // this 'AppInstallation does not exist error' is a race condition that occurs when instantiating the eventsService within the lambda.
+          // it is not a valid error to show to the user.
+          const errorMessage = JSON.parse(error?.message);
```

Comment:
Why would error ever be undefined or null in this catch block?

Reply:
ah yeah I think I originally had escape character on `error?.message?.details` to handle if one of those objects does not exist, though I think you are right we don't need the first escape character since that is just handling and undefined error. Will adjust! Thank you!